aspnetcore/signalr/configuration.md

```diff
@@ -56,7 +56,7 @@ The following table describes options for configuring SignalR hubs:
 
 | Option | Description |
 | ------ | ----------- |
-| `HandshakeTimeout` | If the client doesn't send an initial handshake message within this time interval, the connection is closed. |
+| `HandshakeTimeout` | If the client doesn't send an initial handshake message within this time interval, the connection is closed. This is an advanced setting that should only be modified if the client or network are particularly slow, causing handshake timeout errors. For more detail on the Handshake process, see the [SignalR Hub Protocol Specification](https://github.com/aspnet/SignalR/blob/master/specs/HubProtocol.md) |
```

Comment:
If "causing handshake timeout errors" is a (suggested) requirement for setting the option, drop the comma before "causing." If that phrase is just an example of one thing that might accompany a slow client/network such that the option might be set under other slow client/network conditions, the comma is :+1:.

Either way, it needs a period at the end ... after the *SignalR Hub Protocol Specification* link.

Reply:
I did a bit of a hack job at trying to clarify it. The setting should only be changed if you are seeing handshake timeouts **and** those timeouts are due to a slow network (i.e. you expect delays in transmitting data).

To be perfectly honest, this isn't a setting I'd ever expect a user to need to configure, it's there because we're always hesitant to have hard-coded timeouts. The defaults are just fine in 99.999% of cases.